### PR TITLE
Contact Us component

### PR DESCRIPTION
**Pull request by sooraj1002**
created a contact us component with responsiveness for different window sizes. the form itself also changes according to the size of the window.  the form component can also be reused in other places where a form is required with some minor changes according to use case

there is a dummy api call which can be changed when the actual endpoints when created by the back-end team.

### CODE REVIEW

File: src/Components/Contact Us/Contact.jsx [hunk 1]
```diff
@@ -0,0 +1,24 @@
+import React, { useState } from "react";
+import "./contact.css";
+import Form from "./Form";
+
+const Contact = () => {
+  return (
+    <div className="mainClass">
```
Comment (techsavvyash) [inline src/Components/Contact Us/Contact.jsx]: Change this text to be "Technobyte" centric, we are making the website for our society not tech spardha right now.

File: src/Components/Contact Us/Countries.jsx
Comment (techsavvyash) [inline src/Components/Contact Us/Countries.jsx:1]: What is the point of this file?

File: src/Components/Contact Us/Dropdown.jsx [hunk 2]
```diff
@@ -0,0 +1,34 @@
+import Countries from "./Countries";
+
+const Dropdown = ({ selectedCounty, handleCountyChange }) => {
```
Comment (techsavvyash) [inline src/Components/Contact Us/Dropdown.jsx]: what is the use of "selectedCounty" here if we are redefining "selectedCount*r*y" again inside the func?

File: src/Components/Contact Us/Form.jsx [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+
+  const handleCountyChange = (selectedCountry) => {
+    setSelectedCounty(selectedCountry);
+  };
+
+  const handleSubmit = async (e) => {
+    e.preventDefault();
+
+    try {
+      const response = await fetch("", {
```
Comment (techsavvyash) [inline src/Components/Contact Us/Form.jsx]: create a dummy email, and email the form data to "technobyte@nitkkr.ac.in" for now.

**Review by techsavvyash — CHANGES_REQUESTED** [review]
Please make the suggested changes, and also rename the "Contact Us" folder to either "ContactUs" or "Contact_Us" or "Contact-Us"

File: src/Components/Contact Us/Countries.jsx
Comment (sooraj1002) [reply src/Components/Contact Us/Countries.jsx:1]: this was made so that there would be an option to choose countries from a dropdown menu

**Comment by techsavvyash** [comment]
Also, @sooraj1002 do add relevant screenshots and screen recordings to show how the component looks and that its completely responsive

File: src/Components/Contact Us/Dropdown.jsx (hunk 2 above)
Comment (sooraj1002) [reply src/Components/Contact Us/Dropdown.jsx]: the variable was named wrong. it should have been named newCountry. the intended functionality was to preserve the state between renders.

**Comment by sooraj1002** [comment]
image
image
The emailed form data
image

This is the SR


https://github.com/technobyte-nitkkr/frontier/assets/91770496/4b263f10-20d3-4352-af0b-6f10e6778c77

**Review by techsavvyash — APPROVED** [review]
LGTM! 👍🏼

**Pull request merged**